stateMachine/stateMachineHelpers.js

```diff
@@ -0,0 +1,64 @@
+const { helpers } = require('brave-alert-lib')
```

Comment:
I'd love to see some unit tests for the methods in this file.

Reply:
I actually do have some in testStateMachine, towards the bottom.